src/app/src/test/kotlin/org/imperial/mrc/hint/integration/adr/ADRTests.kt

```diff
@@ -67,6 +67,24 @@ class ADRTests : SecureIntegrationTests()
 
     @ParameterizedTest
     @EnumSource(IsAuthorized::class)
+    fun `can get ADR releases`(isAuthorized: IsAuthorized)
+    {
+        testRestTemplate.postForEntity<String>("/adr/key", getPostEntityWithKey())
+        val datasets = testRestTemplate.getForEntity<String>("/adr/datasets")
+
+        assertSecureWithSuccess(isAuthorized, datasets, null)
+
+        val id = if (isAuthorized == IsAuthorized.TRUE)
+        {
+            val data = ObjectMapper().readTree(datasets.body!!)["data"]
+            data.first()["id"].textValue()
+        }
+        else "fake-id"
+
+        val result = testRestTemplate.getForEntity<String>("/adr/datasets/$id/releases")
+        assertSecureWithSuccess(isAuthorized, result, null)
```

Comment:
You could additionally check the response is actually an array using a similar pattern to the test above i.e:
```kotlin
        if (isAuthorized == IsAuthorized.TRUE)
        {
            val data = ObjectMapper().readTree(result.body!!)["data"]
            assertThat(data.isArray).isTrue
        }
```